protocol-units/execution/opt-executor/src/executor/mod.rs

```diff
@@ -58,13 +58,13 @@ impl Executor {
 		let (_aptos_db, reader_writer) = DbReaderWriter::wrap(AptosDB::new_for_test(
 			&maptos_config.chain.maptos_db_path.clone().context("No db path provided.")?,
 		));
-		let core_mempool = Arc::new(RwLock::new(CoreMempool::new(&node_config)));
+
 		let reader = reader_writer.reader.clone();
 		Ok(Self {
-			block_executor : Arc::new(block_executor),
+			block_executor: Arc::new(block_executor),
 			db: reader_writer,
 			signer,
-			core_mempool,
+			//			core_mempool,
```

Comment:
This will no longer work even if uncommented since the let statement for `core_mempool` is gone.